document/AnnexE.mdk

```diff
@@ -0,0 +1,51 @@
+# Annex E (informative) Relationships among rules, correctness criteria, and policies { - #Annex-E }
```

Comment:
I'm not a big fan of this annex and think it's premature to add this content given specific design issues in flux. If you want to check this in, I wouldn't block, though.